triagebot.toml

```diff
@@ -492,6 +492,13 @@ cc = ["@Nadrieril"]
 message = "Some changes might have occurred in exhaustiveness checking"
 cc = ["@Nadrieril"]
 
+[mentions."library/core/src/intrinsics/simd.rs"]
+message = """
+Some changes occurred to the platform-builtins intrinsics. Make sure all
+codegen backends as well as portable-simd get adapted for the changes.
```

Comment:
maybe phrase this so people creating PRs aren't under the impression that they have to update every backend by themselves? iirc gcc and cranelift aren't supposed to block PRs...

Reply:
Yeah, I think we should update the text here, or make a T-compiler decision if we do want to block PRs. My guess is that we're not ready to block on making the backends work yet.

Reply:
Done